src/main/java/duke/bin/common/Constants.java

```diff
@@ -0,0 +1,9 @@
+package duke.bin.common;
+
+/**
+ * Constants used regularly by the UI.
+ */
+public class Constants {
```

Comment:
Is it better to come up with another name because Constants is pretty misleading?

Reply:
I see, okay removed the class